x-pack/osquerybeat/beater/osquerybeat.go

```diff
@@ -251,14 +254,36 @@ func (bt *osquerybeat) Run(b *beat.Beat) error {
 		}
 	}
 
-	setManagerPayload := func(itypes []string) {
+	setManagerPayload := func() {
```

Comment:
Shall we turn this into a method? This `Run` is becoming quite big.

Reply:
Rewriting the Run for 7.14